Register all segments in a bounding box (#7979)
scalableminds/webknossos: CHANGELOG.unreleased.md

- Upgraded s3 client for improved performance when loading remote datasets. [#7936](https://github.com/scalableminds/webknossos/pull/7936)
- To improve performance, only the visible bounding boxes are rendered in the bounding box tab (so-called virtualization). [#7974](https://github.com/scalableminds/webknossos/pull/7974)
- Added support for reading zstd-compressed zarr2 datasets [#7964](https://github.com/scalableminds/webknossos/pull/7964)

> - Added a feature to register all segments for a given bounding box at once with an action in the bounding boxe's context menu. [#7979](https://github.com/scalableminds/webknossos/pull/7979)

Reviewer: I think, it's `box's` but "of ..." feels more natural.

```suggestion
- Added a feature to register all segments for a given bounding box at once via the context menu of the bounding box. [#7979](https://github.com/scalableminds/webknossos/pull/7979)
```
